Readability.js

```diff
@@ -328,6 +344,12 @@ Readability.prototype = {
       if (baseURI == documentURI && uri.charAt(0) == "#") {
         return uri;
       }
+
+      // If it's data uri, return as it is
```

Comment:
Curious, why is this necessary? I'd expect `new URL()` to cope with this... is it just a performance optimization? If so, it's probably worth mentioning that.

Reply:
No, actually I'm not really sure about the performance. I've worked on this issue in my Go port, and over there Go can't handle data URL so I need to check it first. This is carried on when I port it back to JS.

Reply:
Fixed in [4fd18f4](https://github.com/mozilla/readability/pull/590/commits/4fd18f40d17b6b7ed23e770ff92eca5c0be011c6)